app/helpers/policies_helper.rb

```diff
@@ -14,9 +14,13 @@ def policy_agreement_summary_without_html(policy_member_distance)
   # Returns things like "voted strongly against", "has never voted on", etc..
   def policy_agreement_summary(policy_member_distance)
     if policy_member_distance.nil?
-      "voted <strong>unknown about</strong>".html_safe
+        "voted <strong>unknown about</strong>".html_safe
     elsif policy_member_distance.number_of_votes == 0
-      "has <strong>never voted</strong> on".html_safe
+      if I18n.locale == :uk
```

Comment:
In the English version 3, the words in Ukrainian 1 word, do not know whether or not user-friendly via gettext